lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMPass.cpp

```diff
@@ -999,8 +1001,12 @@ struct ConvertTritonGPUToLLVM
     });
 
     mod.walk([&](triton::gpu::AsyncCommitGroupOp asyncCommitGroupOp) -> void {
+#ifdef USE_ROCM
```

Comment:
This is not strictly related to the test fix. I noticed that asyncCommitGroupOp was generated even though it's ptx specific instruction. It turns out that @alefimov-amd  and another colleague left this intentionally a long time ago as a sign that  insert_slice_async should not be generated at all in amd pipeline. I believe this is too strict, at least for now. We lower insert_slice_async to load and insert_slice, and are able to produce correct code, even though, arguably, insert_slice_async should not be generated on a higher level. This will still enable us to borrow OAI code (for example pipeliner, or whatever piece of code uses insert_slice_async). We can discuss more about this offline.

Reply:
Ah, thanks. I was confused by this as well. I thought we did just lower to load/(non async) insert, but also looked like we didn't want to do that.